docs/_sass/footer.scss

```
@@ -1,304 +1,320 @@
+
+  @media #{$tablet} {
+    -webkit-flex: 1 0 310px;
+    flex: 1 0 310px;
+  }
+
+  ul {
+    list-style: none;
+    margin: 10px auto;
+
+    @media #{$tablet} {
+      max-width: 400px;
+    }
+  }
 }
 
 footer {
-	padding: 50px 0 50px 0;
-	font-size: 1.1em;
-	position: relative;
-	background: #1E2117;
-	color: #fff;
-
-	.getnighthawk-logo{
-		display: flex;
-		a{
-			display:block;
-
-			img{
-				width:5rem;
-				vertical-align: middle;
-			}
-			p{
-				display: inline-block;
-				color:white;
-				font-size: 2rem;
-				font-weight: bolder;
-				margin:0rem 0.2rem;
-			}
-		}
-		a:hover{
-			text-decoration: none;
-
-		}
-	}
-	.copyright-cont{
-
-		display: flex;
-		align-items: center;
-		p{
-			color:white;
-		}
-	}
-	.copyright {
-		font-size: .8em;
-		margin: 0 auto;
-
-		@media #{$tablet} {
-			text-align: center;
-		}
-
-	}
-
-	&,
-	a {
-		color: #999;
-	}
-
-	h2 {
-		font-size: 1.2em;
-		margin: 30px 0;
-		margin: 15px auto;
-		color: #fff;
-		font-weight: bolder;
-	}
-
-	.footer-columns {
-		@extend %flexbox;
-		@include flex-flow(wrap);
-		justify-content: center;
-	}
-
-	a {
-		text-decoration: none;
-
-		&:hover {
-			color: #fff;
-		}
-	}
-
-	.legal-line {
-		width: 100%;
-		padding: 30px 0;
-		margin: 0;
-		background-color: #222527;
-
-		a {
-			font-weight: 600;
-		}
-	}
+  padding: 50px 0 50px 0;
+  font-size: 1.1em;
+  position: relative;
+  background: #1e2117;
+  color: #fff;
+
+  .getnighthawk-logo {
+    display: flex;
+    padding-top: 18px;
+    a {
+      display: block;
+
+      img {
+        width: 8rem;
+        vertical-align: middle;
+      }
+    }
+    a:hover {
+      text-decoration: none;
+    }
+  }
+  .getnighthawk-logo p {
+    display: inline-block;
+    color: white;
+    font-size: 2rem;
+    font-weight: bolder;
+    margin: 0rem 0.5rem;
+    padding-top: 18px;
+  }
+  .copyright-cont {
+    display: flex;
+    align-items: center;
+    font-size: 0.8em;
+    p {
+      color: white;
+    }
+  }
+  .copyright {
+    font-size: 0.8em;
+    margin: 0 auto;
+
+    @media #{$tablet} {
+      text-align: center;
+    }
+  }
+
+  &,
+  a {
+    color: #999;
+  }
+
+  h2 {
+    font-size: 1.2em;
+    margin: 30px 0;
+    margin: 15px auto;
+    color: #fff;
+    font-weight: bolder;
+  }
+
+  .footer-columns {
+    // @extend %flexbox;
+    // @include flex-flow(wrap);
+    justify-content: center;
+  }
+
+  a {
+    text-decoration: none;
+
+    &:hover {
+      color: #fff;
+    }
+  }
+
+  .legal-line {
+    width: 100%;
+    padding: 30px 0;
+    margin: 0;
+    background-color: #222527;
+
+    a {
+      font-weight: 600;
+    }
+  }
 }
 
+@media (max-width: 660px) {
+  .foot-container {
+    flex-direction: column;
+  }
+  .getnighthawk-logo{
+    justify-content: center;
+    display: flex;
+    margin: 2rem 0;
+
+  }
+  .copyright-cont {
+    justify-content: center;
+    margin: 1rem 0rem;
+  }
+  .footer-columns {
+    display: flex;
+    justify-content: center;
+    padding-right: 50px;
```

This should solve the alignment issue mentioned by @vinayaksh42 
```suggestion
    padding-right: 37px;
```